src/ImageManipulation.php

```diff
@@ -1057,4 +1069,204 @@ protected function castDimension($value, $dimension)
+                'height' => $this->getHeight(),
+                'alt' => $this->getTitle(),
+                'src' => $this->getURL()
+            ];
+
+            if ($this->IsLazyLoaded()) {
+                $attributes['loading'] = 'lazy';
+            }
+        }
+
+        $attributes = array_merge($attributes, $this->attributes);
+
+        if (isset($attributes['loading']) && $attributes['loading'] === 'eager') {
+            unset($attributes['loading']);
+        }
+
+        $this->extend('updateAttributes', $attributes);
+
+        return $attributes;
+    }
+
+    /**
+     * Custom attributes to process. Falls back to {@link getAttributes()}.
+     *
+     * If at least one argument is passed as a string, all arguments act as excludes, by name.
+     *
+     * @param array $attributes
+     * @return string
+     */
+    public function getAttributesHTML($attributes = null)
+    {
+        $exclude = null;
+
+        if (is_string($attributes)) {
+            $exclude = func_get_args();
+        }
+
+        if (!$attributes || is_string($attributes)) {
+            $attributes = $this->getAttributes();
+        }
+
+        $attributes = (array) $attributes;
+
+        $attributes = array_filter($attributes, function ($v) {
+            return ($v || $v === 0 || $v === '0');
+        });
+
+        if ($exclude) {
+            $attributes = array_diff_key(
+                $attributes,
+                array_flip($exclude)
+            );
+        }
+
+        // Create markup
+        $parts = [];
+
+        foreach ($attributes as $name => $value) {
+            if ($value === true) {
+                $value = $name;
+            } else {
+                if (is_scalar($value)) {
+                    $value = (string) $value;
+                } else {
+                    $value = json_encode($value);
+                }
+            }
+
+            $parts[] = sprintf('%s="%s"', Convert::raw2att($name), Convert::raw2att($value));
+        }
+
+        return implode(' ', $parts);
+    }
+
+    /**
+     * Determine whether the image should be lazy loaded
+     *
+     * @return bool
+     */
+    public function IsLazyLoaded(): bool
+    {
+        if (Image::getLazyLoadingEnabled() && $this->getIsImage()) {
+            return empty($this->attributes['loading']) || $this->attributes['loading'] === 'lazy';
+        }
+        return false;
+    }
+
+    /**
+     * Set the lazy loading state for this Image
+     * @param mixed $lazyLoad
+     * @return AssetContainer
+     */
+    public function LazyLoad($lazyLoad): AssetContainer
+    {
+        if (!Image::getLazyLoadingEnabled() || !$this->getIsImage()) {
+            return $this;
+        }
+
+        if (is_string($lazyLoad)) {
+            $lazyLoad = strtolower($lazyLoad);
+        }
+
+        $equivalence = [
+            'eager' => [false, 0, '0', 'false', 'eager'],
+            'lazy' => [true, 1, '1', 'true', 'lazy'],
```

Comment:
So, I don't think .LazyLoad(true) will ever actually do anything due to design design to lazy load by default and not make the default value configurable

The only configurable change a dev can make it to globaly disable this functionality

If they do this, then it will fail the `Image::getLazyLoadingEnabled()` conditional and never reach this point

I manually validated this locally testing locally with lazy loading disabled and have a template with $MyImage.LazyLoad(true) - no loading attribute was added

We should just remove this code, and any related code to allows a dev to "add" loading="lazy"

Reply:
If you can disable something, then you need a way to re-enable it later on. e.g.: A developer could have a bit of logic that start by disabling lazy loading on an image and selectively turns it back on later on.